gulpfile.js

```diff
@@ -20,30 +14,33 @@ const rollupCommon = require('@rollup/plugin-commonjs');
 const rollupReplace = require('@rollup/plugin-replace');
 const { terser: rollupTerser } = require('rollup-plugin-terser');
 
+const readJSON = async path => {
+  const content = await fs.readFile(path, 'utf-8');
+  return JSON.parse(content);
+};
+
 function css() {
   const boundSass = gulpSass(sass);
-  return gulp.src('src/css/*.scss', { sourcemaps: true })
```

Comment:
Removed `{ sourcemaps: true }` since we are loading only scss files which don't have sourcemaps